client/src/components/pages/home/quote.tsx

```diff
@@ -1,9 +1,15 @@
 import './quote.css';
+import { FormattedMessage } from "react-intl";
 
 const Quote = () => {
   return (
     <div className="quote">
-      <h1>Quote</h1>
+      <h1>
+        <FormattedMessage
+          id="home.quote"
+          defaultMessage="...working to create a more equitable restaurant ecosystem for all..."
```

Comment:
```suggestion
```

Reply:
We have set the translation package to default to english.

It's best practice to have a single source of truth for data, so that changes only happen in one place.